pyproj/geod.py

```diff
@@ -570,10 +570,11 @@ def geometry_area_perimeter(
         -------
         (float, float):
             The geodesic area (meters^2) and permimeter (meters) of the polygon.
-       """
+        """
         try:
             return self.polygon_area_perimeter(  # type: ignore
-                *geometry.xy, radians=radians,
+                *geometry.xy,
+                radians=radians,
```

Comment:
BTW, the reason those are now put on multiple lines (while they fit on a single) is the trailing comma AFAIU. If you want to keep it on a single line, you can remove the comma (not that it is a big issue though ;))

Reply:
Nice - that is a good tip. Thanks :+1: